docs/standard/serverless-architecture/architecture-approaches/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Serverless Apps. Architecture, Patterns, and Azure Implementation.
+description: Serverless Apps. Architecture, Patterns, and Azure Implementation. | Architecture approaches
+keywords: Serverless, Microservices, .NET, Azure, Azure Functions
```

Comment:
We don't need the "keywords" metadata anymore. (No need to remove them, however).